config/colcon.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: colcon
+method: https://packagecloud.io/dirk-thomas/colcon
+suites: [xenial, bionic, focal, stretch, buster]
+component: main
+architectures: [i386, amd64, armhf, arm64, source]
+filter_formula: Package (% python3-colcon-* )
```

Comment:
```suggestion
filter_formula: (Package (% python3-colcon-argcomplete, $Version (% 0.3.3*))|(Package (% python3-colcon-bash, $Version (% 0.4.2*))|(Package (% python3-colcon-bazel, $Version (% 0.1.0*))|(Package (% python3-colcon-cd, $Version (% 0.1.1*))|(Package (% python3-colcon-cmake, $Version (% 0.2.25*))|(Package (% python3-colcon-cmake, $Version (% 0.2.26*))|(Package (% python3-colcon-common-extensions, $Version (% 0.2.1*))|(Package (% python3-colcon-core, $Version (% 0.6.0*))|(Package (% python3-colcon-coveragepy-result, $Version (% 0.0.8*))|(Package (% python3-colcon-defaults, $Version (% 0.2.5*))|(Package (% python3-colcon-devtools, $Version (% 0.2.2*))|(Package (% python3-colcon-ed, $Version (% 0.1.0*))|(Package (% python3-colcon-lcov-result, $Version (% 0.4.1*))|(Package (% python3-colcon-library-path, $Version (% 0.2.1*))|(Package (% python3-colcon-metadata, $Version (% 0.2.5*))|(Package (% python3-colcon-mixin, $Version (% 0.2.0*))|(Package (% python3-colcon-notification, $Version (% 0.2.13*))|(Package (% python3-colcon-output, $Version (% 0.2.11*))|(Package (% python3-colcon-package-information, $Version (% 0.3.3*))|(Package (% python3-colcon-package-selection, $Version (% 0.2.8*))|(Package (% python3-colcon-package-selection, $Version (% 0.2.9*))|(Package (% python3-colcon-parallel-executor, $Version (% 0.2.4*))|(Package (% python3-colcon-pkg-config, $Version (% 0.1.0*))|(Package (% python3-colcon-powershell, $Version (% 0.3.6*))|(Package (% python3-colcon-python-setup-py, $Version (% 0.2.6*))|(Package (% python3-colcon-recursive-crawl, $Version (% 0.2.1*))|(Package (% python3-colcon-ros, $Version (% 0.3.19*))|(Package (% python3-colcon-ros, $Version (% 0.3.20*))|(Package (% python3-colcon-test-result, $Version (% 0.3.8*))|(Package (% python3-colcon-zsh, $Version (% 0.4.0*))
```

This expanded query includes each package in the colcon repository currently along with it's version (with a wildcard for the package revision). With this change, future updates to packages in the colcon repository would open a PR adding additional packages or changing the version of existing packages to signal a desire to update the ROS bootstrap repository.

Reply:
Is there a way to add newlines into these formula?

Reply:
> Is there a way to add newlines into these formula?

Not in a way that guarantees it will still work as a reprepro-updater configuration since this filter is passed as a command line argument.

Reply:
I hope this works: bd8982c47d35d71f3cead2d5fd708b07721de86f.

Reply:
That seems like it should.